### DISCUSSION TITLE
BattleBoxes Multiplayer Version 1.1.0

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#67] [Releases]
| Version | Changes                      |
| ------- | ---------------------------- |
| 1.0.1  | <ul><li>Fixed achievements not saving</li><li>Fixed no fading between rounds</li></ul> |
| 1.1.0  | <ul><li>Added some sort of bot character (Currently does not pathfind)</li><li>Added "quality" slider to settings (disabled currently)</li><li>Moved drawing from server TPS to client (60 or 30 FPS) and added FPS option</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/56" target="_blank">Fixed transparent scoreboard after playing a game (Issue #56)</a></li><li>Centered winner name text</li><li>Randomized spawnpoints on Platforms and Battlefield</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/58" target="_blank">Fixed force leave on win screen (Issue #58)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/63" target="_blank">Fixed typo in achievements (Issue #63)</a></li><li>Improved camera movement</li><li>Minor optimizations</li></ul> |

<hr /><em>This discussion was created from the release <a href='https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/releases/tag/v1.1.0'>BattleBoxes Multiplayer Version 1.1.0</a>.</em>

**Comment by SampleProvider** [comment]
rrrrrrrrrr
